src/pages/blog/index.en.js

```diff
@@ -9,6 +9,8 @@ export const frontmatter = {
   id:  '02',
```

Comment:
I have to redesign the blog index page: I should create a blog-index template and inject all the data in the frontmatter of the markdown.

Reply:
see this issue: https://github.com/kalwalt/gatsby-starter-i18n-bulma/issues/31